src/python_vali/src/VALI.cpp

```diff
@@ -378,6 +378,12 @@ PYBIND11_MODULE(_python_vali, m) {
       .def(py::init<>())
       .def(py::init<ColorSpace, ColorRange>(), py::arg("color_space"),
            py::arg("color_range"))
+      .def(py::init([](const PyDecoder& decoder) {
```

Comment:
Same thing here.
Please consider taking `py::object` and check for `ColorSpace` and `ColorRange` attributes.

Reply:
It means that as soon as someone adds them, developers will be able to make conversion context from `PyNvEncoder`.